Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -368,6 +368,9 @@ def search_slack_users(users: Union[list, str]) -> list:
     if not isinstance(users, list):
         users = [users]
 
+    # Filter NoneType elements from the user list
+    users = list(filter(lambda x: x is not None, users))
```

Comment:
`x is not None` that may cause that x = '' will be included in the list.
You should change it to `lambda x: x`
Add unit test please